flag/service/security/security.go

```diff
@@ -0,0 +1,15 @@
+package security
+
+type Security struct {
+	Letsencrypt      Letsencrypt
```

Comment:
Is this still needed (given removal of Let's Encrypt)?

Reply:
Well, we will need it for oauth2-proxy so I was thinking it was not really useful to remove it

Reply:
Turns out we do not now so I removed it